src/KafkaJanitor.IntegrationTests/CapabilityCreatedDeletedScenario.cs

```diff
@@ -0,0 +1,147 @@
+            {
+                TIKA_API_ENDPOINT = "http://localhost:3000/"
+            }));
+        }
+
+        private async Task Then_a_service_account_is_created(Capability capability)
+        {
+            serviceAccount = await _tikaClient.ServiceAccounts.CreateAsync(new ServiceAccountCreate {
+                name = $"{capability.Name}_sa",
+                description = "Creating during CapabilityCreatedDeletedScenario"
+                });
+        }
+
+        private async Task And_acls_to_create_read_write_to_topics_under_prefix(Capability capability, ServiceAccount serviceAccount)
+        {
+            var serviceAccountId = Convert.ToInt64(serviceAccount.Id);
+            // Topic
+            await _tikaClient.Acls.CreateAsync(new AclCreateDelete(serviceAccountId, true, "WRITE", capability.Name));
+            await _tikaClient.Acls.CreateAsync(new AclCreateDelete(serviceAccountId, true, "CREATE", capability.Name));
+            await _tikaClient.Acls.CreateAsync(new AclCreateDelete(serviceAccountId, true, "READ", capability.Name));
+            await _tikaClient.Acls.CreateAsync(new AclCreateDelete(serviceAccountId, true, "DESCRIBE", capability.Name));
+            await _tikaClient.Acls.CreateAsync(new AclCreateDelete(serviceAccountId, true, "DESCRIBE-CONFIGS", capability.Name));
+
+            // ConsumerGroup
+            await _tikaClient.Acls.CreateAsync(new AclCreateDelete(serviceAccountId, true, "WRITE", "", capability.Name));
+            await _tikaClient.Acls.CreateAsync(new AclCreateDelete(serviceAccountId, true, "CREATE", "", capability.Name));
+            await _tikaClient.Acls.CreateAsync(new AclCreateDelete(serviceAccountId, true, "READ", "", capability.Name));
+            
+            // DENY
+            await _tikaClient.Acls.CreateAsync(new AclCreateDelete(serviceAccountId, false, "alter"));
+            await _tikaClient.Acls.CreateAsync(new AclCreateDelete(serviceAccountId, false, "alter-configs"));
+            await _tikaClient.Acls.CreateAsync(new AclCreateDelete(serviceAccountId, false, "cluster-action"));
+            await _tikaClient.Acls.CreateAsync(new AclCreateDelete(serviceAccountId, false, "create", "'*'"));
+        }
+
+        private async Task When_a_capability_is_deleted()
+        {
+            var kafkaClient = new KafkaClient();
+            var capability = new Capability
+            {
+                Id = "1e0a8f85-de38-42ef-a4f4-87c3b4f9a5f9",
+                Name = "devx-acltest4"
+            };
+
+            var message = new
+            {
+                CapabilityId = capability.Id,
+                CapabilityName = capability.Name
+            };
+
+            await kafkaClient.SendMessageAsync(message, "capability_deleted");
+        }
+
+        private async Task Then_the_connected_acls_are_deleted(Capability capability, ServiceAccount serviceAccount)
+        {
+            var serviceAccountId = Convert.ToInt64(serviceAccount.Id);
```

Comment:
This should live somewhere else

Reply:
So it shall. https://app.clubhouse.io/devex/story/1371/change-kafka-janitor-integration-tests-to-use-services-startup-cs-for-tests-instead-of-the-clients-directly